float32[] position 
float32[] orientation
float32[] transf_matrices
float32[] joint_torques
bool is_fallen